param/thrust_models/thrust_model_2.0.yaml

```diff
@@ -1,6 +1,6 @@
 # Generated on 2018-05-18 21:21:38.749113
 
-response_lag: 0.03184030109665681
+response_lag: 0.0 #0.03184030109665681
```

Comment:
whats going on with this?

Reply:
We can't account for response lag because state estimation is not extrapolated into the future.